config/base/src/lib.rs

```diff
@@ -356,24 +380,27 @@ pub mod runtime_upgrades;
 pub mod view {
     //! Module for view related traits and structs
 
-    /// Marker trait to set default value `IS_HAS_VIEW` to `false`
+    /// Marker trait to set default value [`IsInstanceHasView::IS_INSTANCE_HAS_VIEW`] to `false`
     pub trait NoView {
         /// [`Self`] doesn't implement [`HasView`]
         const IS_HAS_VIEW: bool = false;
```

Comment:
Maybe it would be more idiomatic to use "strong type" instead of a `bool`:

```rust
enum ViewExistence {
  Exists,
  Absent
}
```

Reply:
The biggest benefit of this approach is that you need to use a `match` or an `if let` instead of `if`. Users of the library don't expect a yes/no question to be answered with an enum, unless there's a good reason. using 
```rust 
enum HasView {
  Has(View),
  Not
}
```
would be better, but I don't see how to do it yet.